src/Sylius/Bundle/CoreBundle/Form/Type/CartItemType.php

```diff
@@ -34,7 +34,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         parent::buildForm($builder, $options);
 
-        if (isset($options['product']) && $options['product']->hasVariants()) {
+        if (isset($options['product']) && $options['product']->hasVariants() && !$options['product']->isSimple()) {
```

Comment:
I added it back in because it was the cause of some tests failing on the cart summary page